fastlane/lib/fastlane/actions/ipa.rb

```diff
@@ -69,10 +80,7 @@ def self.run(params)
           ENV[SharedValues::IPA_OUTPUT_PATH.to_s] = absolute_ipa_path # for deliver
           ENV[SharedValues::DSYM_OUTPUT_PATH.to_s] = absolute_dsym_path
 
-          UI.important("You are using legacy `shenzhen` to build your app")
-          UI.important("It is recommended to upgrade to `gym`")
-          UI.important("To do so, just replace `ipa(...)` with `gym(...)` in your Fastfile")
-          UI.important("https://github.com/fastlane/fastlane/tree/master/gym")
+          deprecation_warning
```

Comment:
This is nice, we now show the message twice. Once this PR gets merged and release, let's get rid of this as well, and have `ipa` be an alias for `gym`, so we get rid of `shenzhen` completely 👍